src/config/samples/policy.yaml

```diff
@@ -57,6 +57,11 @@ spec:
       openSearchAddr: "https://opensearch-cluster-master.default:9200"
       openSearchUser: "admin"
       openSearchPasswd: "admin"
+      governor:
+        enabled: true
+        clusterId: "65a03970-c53a-4ba1-8d1f-42c9f95d2761"
+        url: "app-catalog.vmware.com/api"
```

Comment:
Is the exporter handles both https and Http?

Reply:
Needs to be discussed with Engineering productivity team.